revolt/channel.py

```diff
@@ -54,6 +58,8 @@ def __init__(self, data: GroupDMChannelPayload, state: State):
         self.owner = state.get_user(data["owner"])
 
 class TextChannel(Channel, Messageable):
+    __slots__ = ("name", "description", "last_message", "last_message_id") # "server" is already an attribute of "Channel" class
```

Comment:
This comment isnt needed
```suggestion
    __slots__ = ("name", "description", "last_message", "last_message_id")
```